src/services/blocks/BlocksService.ts

```diff
@@ -212,6 +229,7 @@ export class BlocksService extends AbstractService {
 				success: defaultSuccess,
 				// paysFee overrides to bool if `system.ExtrinsicSuccess|ExtrinsicFailed` event is present
 				paysFee: null as null | boolean,
+				docs: extrinsicDocs ? extrinsic.meta.documentation : undefined,
```

Comment:
Elsewhere you have `const eventDocsArg = eventDocs === 'true';` which implies that this should be a `bool`, so I'd expect this to be `docs: extrinsicDocs ? extrinsic.meta.documentation : false,`. Or why do you prefer `undefined` here?

Reply:
AFAIU This is setting the _final value_ itself inside a `{}`, thus the field `docs` will be set either to `extrinsic.meta.documentation` or `undefined`.

Reply:
The reason I chose `undefined` here is that when a property (in this case `docs`) is set to undefined in a response body (in this case a plain-old-javascript-object), the response body will omit that field. If we wanted to keep the field in even if they did not want docs I think `null` would make the most sense.

I think have `docs` set to `false` when the user does not opt in to them might be confusing because the inverse would be having an array of strings for `docs`, not `true`.